src/lib/components/Swap/Input.tsx

```diff
@@ -27,31 +27,48 @@ interface InputProps {
 }
 
 export default function Input({ disabled }: InputProps) {
-  const input = useAtomValue(inputAtom)
-  const setValue = useUpdateInputValue(inputAtom)
-  const setToken = useUpdateInputToken(inputAtom)
-  const balance = mockCurrencyAmount
+  const {
+    currencyBalances: { [Field.INPUT]: balance },
+    currencyAmounts: { [Field.INPUT]: inputCurrencyAmount },
+  } = useSwapInfo()
+  const inputUSDC = useUSDCValue(inputCurrencyAmount)
+
+  const [typedInputAmount, updateTypedInputAmount] = useSwapAmount(Field.INPUT)
```

Comment:
Can these be called `swapInputAmount`/`updateSwapInputAmount` to match the hook?